It's not required to link with libssh_threads anymore since libssh 0.8

Index: x2goclient.pro
--- x2goclient.pro.orig
+++ x2goclient.pro
@@ -165,7 +165,7 @@ else:win32 {
 }
 else {
   # For backwards-compatibility.
-  LIBS += -lssh -lssh_threads
+  LIBS += -lssh
 }
 
 win32:LIBS += -lAdvAPI32 -lshell32 -lUser32
@@ -203,7 +203,7 @@ else:message("No translation files in project")
 
 TEMPLATE = app
 DEPENDPATH += .
-INCLUDEPATH += src
+INCLUDEPATH += src $(X11BASE)/include
 RESOURCES += res/resources.qrc
 
 exists(res/txt/git-info) {
@@ -220,6 +220,10 @@ linux|linux-* {
   message("building $$TARGET via $$QMAKESPEC with ldap and cups")
   LIBS += -lldap -lcups -lX11 -lXpm
   DEFINES += __linux__
+}
+openbsd-* {
+    message("building $$TARGET with ldap and cups")
+    LIBS += -lldap -lcups -lX11 -lXpm
 }
 x2go_linux_static {
   message("linking all libs statically")
